### PR TITLE
Update with-universal-configuration examples

### LINKED ISSUE #4498: Update with-universal-configuration example
# Feature request

## Is your feature request related to a problem? Please describe.
It is confusing for new users to get environment variables set up because there are examples that do not work, and there is an officially supported way of doing environment variables now

https://github.com/zeit/next.js#exposing-configuration-to-the-server--client-side

## Describe the solution you'd like
Remove `with-universal-configuration` from the repo

## Describe alternatives you've considered
Open to keeping them if someone shows a unique use case.

### PR DESCRIPTION
**Pull request by davscro**
resolves #4498

### CODE REVIEW

**Comment by davscro** [comment]
hi @timneutkens , what is the status of this pull request? Let me know if I have to adjust something.
Thanks in advance 😉

File: examples/with-universal-configuration-runtime/pages/index.js [hunk 1]
```diff
@@ -1,17 +1,19 @@
 import React from 'react'
-import env from '../lib/env'
+import getConfig from 'next/config'
```
Comment (timneutkens) [inline examples/with-universal-configuration-runtime/pages/index.js]: it seems like 2 PRs are mixed here

**Comment by timneutkens** [comment]
Actually, the easiest way to go is doing this:

Change `this.props.url` here:

https://github.com/zeit/next.js/blob/canary/server/document.js#L99

To:

```
{(children || []).map((child) => {
  if(child.type === 'title') {
     // implement warning logic here
  }
  return child
})}
```

**Comment by davscro** [comment]
title PR is here #4596

File: examples/with-universal-configuration-runtime/pages/index.js (hunk 1 above)
Comment (davscro) [reply examples/with-universal-configuration-runtime/pages/index.js]: I resolved that, now they are seperated...

**Pull request closed without merging**